aws/aws-ecr-event-forwarder-enhanced/dynatrace_aws_event_forwarder_enhanced_template.yaml

```diff
@@ -0,0 +1,192 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: Template for Dynatrace AWS Image Scan Event Forwarder - Enhanced Scanning.
+
+Metadata:
+  License:
+    Description: |
+      Copyright 2024 Dynatrace LLC
+
+      Licensed under the Apache License, Version 2.0 (the "License");
+      you may not use this file except in compliance with the License.
+      You may obtain a copy of the License at
+
+          https://www.apache.org/licenses/LICENSE-2.0
+
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+
+Parameters:
+  AwsSecretArn:
+    Description: Arn of the created secret upfront.
+    Type: String
+
+  AwsSecretKeyName:
+    Description: Name of the secret key.
+    Default: "DYNATRACE_OPENPIPELINE_INGEST_API_TOKEN"
+    Type: String
+
+  AwsInspectorResourceTypes:
+    Description: Defines for which resource types the lambda function sends events to Dynatrace. Multiple resource types can be provided with a comma separated list.
+    Default: "AWS_ECR_CONTAINER_IMAGE"
+    Type: String
+
+  DynatraceDomain:
```

Comment:
Hey, from my experience, having an allowed regular expression here can save us a few RFA's in the future. Can you please take a look at [these properties](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html#parameters-section-structure-properties) and add AllowedPattern property?

Reply:
Added AllowedPattern and ConstraintDescription for the DynatraceDomain parameter.